Pipfile

```diff
@@ -24,13 +27,14 @@ requests = "==2.17.3"
 structlog = "==17.2.0"
 flask-httpauth = "==3.2.3"
 ras-common-utils = {git = "https://github.com/ONSdigital/ras-common-utils.git", ref = "0.0.7"}
+retrying = "*"
 
 
 [dev-packages]
 
 tox = "*"
 coverage = "*"
-flake8 = "*"
+"flake8" = "*"
```

Comment:
why the quotes?

Reply:
It was automatically added when pipenv install. I can remove

Reply:
Leave it in. It's Pipenv normalising the package names. Those with numbers or special characters are intentionally quoted probably from checking against PyPI real names.